packages/backend/src/controllers/organizationController.ts

```diff
@@ -175,13 +176,25 @@ export class OrganizationController extends BaseController {
     async getOrganizationMembers(
         @Request() req: express.Request,
         @Query() includeGroups?: number,
+        @Query() pageSize?: number,
+        @Query() page?: number,
+        @Query() searchQuery?: string,
     ): Promise<ApiOrganizationMemberProfiles> {
         this.setStatus(200);
+        let paginateArgs: KnexPaginateArgs | undefined;
```

Comment:
we might want to reuse this on any endpoint that implements pagination, so it would be nice to extract this into a single function . 

But we can do it later, once this is reused.